templates/patterns/featured_item/featured_item.ui_patterns.yml

```diff
@@ -5,17 +5,15 @@ featured_item:
     default:
       label: "Default"
       description: "Default type of the featured item."
-    extended:
-      label: "Extended"
-      description: "External type of the featured item."
+    tile:
+      label: "Tile"
+      description: "Simple type of the featured item."
```

Comment:
The scope of the ticket is to port our previous featured item pattern to ECL 2.x, so we shouldn't change the pattern definition as much as possible.

Let's keep the variants as they were and fin a way to render them using ECL 2.x so they look as close as they can to the previous one (even not necessarily by using a card component on both of them, maybe the card component can fit only the default variant).